### ISSUE TITLE
Improve lore2sphinx buildbot results highlevel docs in `projects/core/howto/`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4873]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4873|
|Type|enhancement|
|Created|2011-02-11 05:38:04Z|
|Branch|https://github.com/twisted/twisted/tree/highlevel-4873|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the basics and application howtos

<details><summary>Searchable metadata</summary>

```
trac-id__4873 4873
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_highlevel_4873 branches/highlevel-4873
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402684000000 1297402684000000
changetime__1300077456000000 1300077456000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30678]) Branching to 'highlevel-4873'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30679]) Merge the basics and application parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4873](https://github.com/twisted/twisted/issues/4873)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_7"></a>|@lvh commented|
|-|-|

The actual problem with basics.xhtml will be fixed manually in [#4941](https://github.com/twisted/twisted/issues/4941).

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn set owner to @lvh|
|-|-|

Merge it!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_9"></a>|@lvh set status to `closed`|
|-|-|

(In [31004]) Improve lore2sphinx buildbot results highlevel docs in projects/core/howto/.


Author: screwtape, jdb
Reviewer: khorn
Fixes: [#4873](https://github.com/twisted/twisted/issues/4873)
  
"Makes Sphinx looks gooder." -- JP

There are problems left, for which there is [#4941](https://github.com/twisted/twisted/issues/4941).
